vhdbuilder/packer/components.json

```diff
@@ -319,25 +319,25 @@
       "downloadLocation": "/opt/cni/downloads",
       "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
       "versions": [
-        "1.5.11",
-        "1.4.43.1"
+        "1.5.13",
```

Comment:
should we skip updating 1.5.11? The rp toggle still uses 1.5.11 and I don't think it was showing up in trivvy scans.

Reply:
We should keep it, I'll add it back